financial-connections/src/main/java/com/stripe/android/financialconnections/features/success/SuccessViewModel.kt

```diff
@@ -194,18 +96,16 @@ internal class SuccessViewModel @Inject constructor(
 }
 
 internal data class SuccessState(
+    // Just enabled on Compose Previews: allows to preview the post-animation state.
+    val overrideAnimationForPreview: Boolean = false,
```

Comment:
Is there a way for us to avoid this, e.g. by using `LocalInspectionMode` in the composable? I don’t want us to leak this into the data model.

Reply:
Since we also want to see the non-overriden state in previews `LocalInspectionMode` is not very helpful. Instead I moved the override out of the check and made it internal to the component: https://github.com/stripe/stripe-android/pull/7761/commits/0658c1bc98cee93654ad0e7c5ae55ebd0377bc5a